types/react-transition-group/react-transition-group-tests.tsx

```diff
@@ -178,38 +174,53 @@ const Test: React.StatelessComponent = () => {
                     appear
                     enter
                     exit
-                    timeout={ 500 }
-                    addEndListener={ handleEndListener }
-                    onEnter={ handleEnter }
-                    onEntering={ handleEnter }
-                    onEntered={ handleEnter }
-                    onExit={ handleExit }
-                    onExiting={ handleExit }
-                    onExited={ handleExit }
+                    timeout={500}
+                    addEndListener={handleEndListener}
+                    onEnter={handleEnter}
+                    onEntering={handleEnter}
+                    onEntered={handleEnter}
+                    onExit={handleExit}
+                    onExiting={handleExit}
+                    onExited={handleExit}
                     classNames="fade"
                 >
-                    <div>{ "test" }</div>
+                    <div>{'test'}</div>
+                </CSSTransition>
+
+                <CSSTransition
+                    timeout={{ enter: 500, exit: 500 }}
+                    classNames={{
+                        appear: 'fade-appear',
+                        appearActive: 'fade-active-appear',
+                        appearDone: 'fade-done-appear',
+                        enter: 'fade-enter',
+                        enterActive: 'fade-active-enter',
+                        enterDone: 'fade-done-enter',
+                        exit: 'fade-exit',
+                        exitActive: 'fade-active-exit',
+                        exitDone: 'fade-done-exit',
+                    }}
+                >
+                    <div>{'test'}</div>
+                </CSSTransition>
+
+                <CSSTransition timeout={100}>
+                    <div>{'test'}</div>
                 </CSSTransition>
 
                 <CSSTransition
-                    timeout={ { enter : 500, exit : 500 } }
-                    classNames={ {
-                        appear: "fade-appear",
-                        appearActive: "fade-active-appear",
-                        appearDone: "fade-done-appear",
-                        enter: "fade-enter",
-                        enterActive: "fade-active-enter",
-                        enterDone: "fade-done-enter",
-                        exit: "fade-exit",
-                        exitActive: "fade-active-exit",
-                        exitDone: "fade-done-exit",
-                    } }
+                    timeout={500}
+                    onEnter={(node, isAppearing) => {}}
```

Comment:
How about testing the arguments' types as follows?

```suggestion
                    onEnter={handleEnter}
```

Reply:
Good idea, changed.